modules/core/sfx.js

```diff
@@ -145,7 +146,18 @@ yargs(hideBin(process.argv))
         })
       } catch (e) {
         consola.error(e)
-
+        process.exit(1)
+      }
+    },
+  })
+  .command({
+    command: 'make module',
```

Comment:
Thats not a proper way how to define a command with arguments. Look at documentation of yargs. make is still the command, but it takes argument what to make (for now only possible option is module).